src/developer-guide/nodejs-api.md

```diff
@@ -917,14 +916,14 @@ ruleTester.run("my-rule", myRule, {
 
 ---
 
-[configuration object]: ../user-guide/configuring/
-[builtin-formatters]: https://eslint.org/docs/user-guide/formatters/
+[configuration object]: ../user-guide/configuring/index
```

Comment:
This URL shouldn't have `index`. Also, `[configuration object]` link is unused on the translated page because the text is translated so it appears as `[配置对象]` (check `options.baseConfig` and `options.overrideConfig` list items) so we should either add the URL there or update the text here.